### PR TITLE
Advanced login

### PR DESCRIPTION
**Pull request by ducquanghuynh0111**
ready
nh ch p man hinh 2018-12-10 luc 10 29 02

### CODE REVIEW

**Comment by ducquanghuynh0111** [comment]
ready

**Comment by nvtanh** [comment]
commented

2 commit

**Comment by ducquanghuynh0111** [comment]
ready

File: Gemfile.lock [hunk 1]
```diff
@@ -50,8 +50,13 @@ GEM
     ast (2.4.0)
     autoprefixer-rails (9.4.0)
       execjs
+<<<<<<< HEAD
     bindex (0.5.0)
     bcrypt (= 3.1.12)
+=======
+    bcrypt (3.1.12)
+    bindex (0.5.0)
+>>>>>>> Advanced login
```
Comment (chien-pv) [inline Gemfile.lock]: (tat) fix lại đi em

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user&.authenticate params[:session][:password]
+      log_in user
+      remember user
+      redirect_to user
+    else
+      flash.now[:danger] = t ".error_messages"
+      render "new"
```
Comment (chien-pv) [inline app/controllers/sessions_controller.rb]: :new

**Comment by chien-pv** [comment]
commented

**Comment by ducquanghuynh0111** [comment]
ready

**Comment by chien-pv** [comment]
merged

**Pull request merged**